### ISSUE TITLE
[Feature request]: Install terminal auto-completion

### ISSUE THREAD

**Issue by gustavonmartins** [skim-rs/skim#315]
I have installed skim and prefer it over fzf, but I miss being able of pressing CTRL+T for auto completion in terminal, and CTRL+R for history auto complication and ALT+C for changing directory.

I have installed both fzf and skim thru brew, and after finishing the installation of fzf in brew it warns me that I can add those auto completions by running
/home/linuxbrew/.linuxbrew/opt/fzf/install

It would be very nice to provide the same feature for skim.

Thanks a lot!

**Comment by murlakatamenka** [comment]
That's seems to be a packaging issue, not of `skim` itself. For example, on Arch Linux skim comes with those conveniences:

https://www.archlinux.org/packages/community/x86_64/skim/ -> check package contents -> `completion.zsh` / `key-bindings.zsh`. I have it sourced in my .rc files and I can use key bindings like `Ctrl + T` to pick files and `Ctrl + R` to find a command in shell history. Maybe such files are present in brew package too?

**Comment by namoscagnm** [comment]
Yes, those files are installed, but I have to source them manually as you said. In fzf, after installing it can source it automatically for you. If you are a power user its ok to source it manually, but for most of people the auto sourcing of fzf is nicer.

I installed it thru pacman now in Manjaro and unfortunately the situation is the same

**Comment by keblek** [comment]
I am on ubuntu and installed through 'cargo install' it would be nice if there was a short section in the readme for how to turn these features on

**Comment by tombh** [comment]
Another point is that it's not mentioned anywhere in the docs that this repo comes with shell bindings and completions.

**Comment by stevenroose** [comment]
Yeah and the install script also doesn't have the option to do that like fzf's has. I can't really get them to work manually, so I don't think I could contribute them becoming part of the install script.
